.semaphore/semaphore.yml

```diff
@@ -7,7 +7,7 @@ version: v1.0
 name: build-test-release
 agent:
   machine:
-    type: s1-prod-ubuntu20-04-amd64-1
+    type: s1-prod-ubuntu20-04-amd64-2
```

Comment:
curious, is this the recommended way to scaling up in semaphore?

Reply:
it seems so
https://confluentinc.atlassian.net/wiki/spaces/TOOLS/pages/2820542346/Semaphore+Self+Hosted+Agents

Reply:
yeah, that is the link 👍